addons/binding/org.openhab.binding.autelis/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,568 @@
+			<channel id="equipment-circuit8" typeId="equipment-circuit8" />
+			<channel id="equipment-circuit9" typeId="equipment-circuit9" />
+			<channel id="equipment-circuit10" typeId="equipment-circuit10" />
+			<channel id="equipment-feature1" typeId="equipment-feature1" />
+			<channel id="equipment-feature2" typeId="equipment-feature2" />
+			<channel id="equipment-feature3" typeId="equipment-feature3" />
+			<channel id="equipment-feature4" typeId="equipment-feature4" />
+			<channel id="equipment-feature5" typeId="equipment-feature5" />
+			<channel id="equipment-feature6" typeId="equipment-feature6" />
+			<channel id="equipment-feature7" typeId="equipment-feature7" />
+			<channel id="equipment-feature8" typeId="equipment-feature8" />
+			<channel id="equipment-feature9" typeId="equipment-feature9" />
+			<channel id="equipment-feature10" typeId="equipment-feature10" />
+			<channel id="temp-poolht" typeId="temp-poolht" />
+			<channel id="temp-spaht" typeId="temp-spaht" />
+			<channel id="temp-htstatus" typeId="temp-htstatus" />
+			<channel id="temp-poolsp" typeId="temp-poolsp" />
+			<channel id="temp-spasp" typeId="temp-spasp" />
+			<channel id="temp-pooltemp" typeId="temp-pooltemp" />
+			<channel id="temp-spatemp" typeId="temp-spatemp" />
+			<channel id="temp-airtemp" typeId="temp-airtemp" />
+			<channel id="temp-tempunits" typeId="temp-tempunits" />
+			<channel id="temp-htpump" typeId="temp-htpump" />
+			<channel id="pump-pump1" typeId="pump-pump1" />
+			<channel id="pump-pump2" typeId="pump-pump2" />
+			<channel id="pump-pump3" typeId="pump-pump3" />
+			<channel id="pump-pump4" typeId="pump-pump4" />
+			<channel id="pump-pump5" typeId="pump-pump5" />
+			<channel id="pump-pump6" typeId="pump-pump6" />
+			<channel id="pump-pump7" typeId="pump-pump7" />
+			<channel id="pump-pump8" typeId="pump-pump8" />
+			<channel id="chlor-chloren" typeId="chlor-chloren" />
+			<channel id="chlor-poolsp" typeId="chlor-poolsp" />
+			<channel id="chlor-spasp" typeId="chlor-spasp" />
+			<channel id="chlor-salt" typeId="chlor-salt" />
+			<channel id="chlor-super" typeId="chlor-super" />
+			<channel id="chlor-chlorerr" typeId="chlor-chlorerr" />
+			<channel id="chlor-chlorname" typeId="chlor-chlorname" />
+			<channel id="lightscmd" typeId="lightscmd" />
+		</channels>
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<context>network-address</context>
+				<label>Host or IP</label>
+				<description>The host name or IP address of the Autelis Controller.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65335"  required="true">
+				<context>network-address</context>
+				<label>Port</label>
+				<description>The port the Autelis Controller is listening on.</description>
+				<default>80</default>
+				<required>true</required>
+			</parameter>
+			<parameter name="user" type="text" required="true">
```

Comment:
Does this controller always require credentials? Or is it also possible to configure it without need to login or are there e.g. some default credentials? (both options would make the parameter optional)